### ISSUE TITLE
Compile Errors with IDA 7.4 SDK

### ISSUE THREAD

**Issue by alexander-pick** [avast-tl/retdec-idaplugin#53]
I was trying to compile the plugin for IDA 7.4 and got multiple errors:

"inf" is not declared
"WOPN_TAB" is not declared
"display_widget"is not accepting a single param

If required I can attach a build log but it is german.

**Comment by alexander-pick** [comment]
WOPN_TAB and the display_widget error can be solved by changing line 124 of code_viewer.h to 

```
WOPN_DP_TAB
```

The inf thing seems to be more complicated, I had top add to "idaplugin/defs.h".

```
#ifdef NO_OBSOLETE_FUNCS
#undef NO_OBSOLETE_FUNCS
#endif
```

This gave me some new errors, is_32bit and is_64bit are missing. This can be fixed by changing line 407 and 534 in idaplugin.cpp. You need to replace 

```
inf.is_32bit
inf.is_64bit 
```
with 
```
inf_is32bit
inf_is64bit
```

After these changes the plugin compiles fine, no guarantee yet it works :)

**Comment by s3rvac** [comment]
@PeterMatula Could you please take a look?

**Comment by alexander-pick** [comment]
I submitted a pull request with the fixes.

**Comment by s3rvac** [comment]
Thanks! @PeterMatula will review the PR (#54).

**Comment by s3rvac** [comment]
Fixed in #54, closing.
